pkg/cmd/experimental/tokens/whoami.go

```diff
@@ -29,14 +37,13 @@ func NewCmdWhoAmI(clientCfg *clientcmd.Config) *cobra.Command {
 	return cmd
 }
 
-func whoami(token string, clientCfg *clientcmd.Config) {
-	osCfg := clientCfg.OpenShiftConfig()
+func whoami(token string, clientCfg *kclient.Config, cmd *cobra.Command) {
 	// This will be pulled out of the auth config file after https://github.com/GoogleCloudPlatform/kubernetes/pull/2437
```

Comment:
Not bound automatically in a first try so I'm adding a TODO to investigate later.

----- Original Message -----
> > @@ -29,14 +37,13 @@ func NewCmdWhoAmI(clientCfg *clientcmd.Config)
> > *cobra.Command {
> >  	return cmd
> >  }
> >  
> > -func whoami(token string, clientCfg *clientcmd.Config) {
> > -	osCfg := clientCfg.OpenShiftConfig()
> > +func whoami(token string, clientCfg *kclient.Config, cmd *cobra.Command) {
> >  	// This will be pulled out of the auth config file after
> >  	https://github.com/GoogleCloudPlatform/kubernetes/pull/2437
> 
> Go ahead and do this cleanup while you're here.  The token from the bound
> clientConfig should be used.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/830/files#r24107268